core/modules/url/router/beforeEach.ts

```diff
@@ -12,7 +12,7 @@ import config from 'config'
 import { RouterManager } from '@vue-storefront/core/lib/router-manager'
 
 export const UrlDispatchMapper = async (to) => {
-  const routeData = await store.dispatch('url/mapUrl', { url: to.fullPath, query: to.query })
+  const routeData = await store.dispatch('url/mapUrl', { url: to.path, query: to.query })
```

Comment:
if we're not giving a `fullPath` to the router we should pass it as a separate argument (like with `query`); it may be useful for devs; I get the idea to change from `fullPath` to `path` because it's safer (as most of the usersr assume that this is, in fact, a `path` - it was with hash = query = surprise ;))

Reply:
@pkarw I disagree. fullPath in SSR is the same as path because the hash is omitted. If we include it then it will create an inconsistency between SSR and CSR. I can't imagine a case when you care about hash during routing

Reply:
OK we can keep it as it is; please resolve the conflicts